integtest/src/main/java/org/corfudb/universe/scenario/Fixtures.java

```diff
@@ -0,0 +1,104 @@
+package org.corfudb.universe.scenario;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import lombok.Builder;
+import lombok.Getter;
+import org.corfudb.universe.scenario.Scenario.Fixture;
+import org.corfudb.universe.service.Service.ServiceParams;
+import org.slf4j.event.Level;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import static lombok.Builder.Default;
+import static org.corfudb.universe.cluster.Cluster.ClusterParams;
+import static org.corfudb.universe.node.CorfuServer.Mode;
+import static org.corfudb.universe.node.CorfuServer.Persistence;
+import static org.corfudb.universe.node.CorfuServer.ServerParams;
+
+public interface Fixtures {
+
+    @Builder
+    @Getter
+    class SingleServerFixture implements Fixture<ServerParams> {
+        @Default
+        private final int port = 9000;
+        @Default
+        private final Mode mode = Mode.SINGLE;
+
+        @Override
+        public ServerParams data() {
+            return ServerParams.builder()
+                    .mode(mode)
+                    .logDir("/tmp/")
+                    .logLevel(Level.TRACE)
+                    .persistence(Persistence.MEMORY)
+                    .port(port)
+                    .timeout(Duration.ofMinutes(5))
+                    .pollPeriod(Duration.ofMillis(50))
+                    .workflowNumRetry(3)
+                    .build();
+        }
+    }
+
+    @Builder
+    @Getter
+    class MultipleServersFixture implements Fixture<ImmutableList<ServerParams>> {
+        @Default
+        private final int numNodes = 3;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)